loader/0025-pr-policy.md

```diff
@@ -0,0 +1,68 @@
+- **Hands-off** -- Every Loader team member shouldn't have to approve every PR. The team as a whole has a responsibility to ensure PRs are reviewed in a timely manner, but no person should have a responsibility to view every change.
+
+It is **not a goal** of this PR to be fool-proof from bad actors within the Loader team abusing the system. Being on any team in Quilt requires trust, and bad actors should be removed instead of worked around.
+
+# Explanation
+## Initial Review Process
+A Pull Request is considered to have been approved when all of the following are true:
+- The Pull Request has received 2 approvals by members of the Loader team.
+- The Pull Request has no changes requested reviews by any members of the Loader team
+- Nobody in the Loader team has has their review requested on the Pull Request
+    - This includes someone requesting the review of themselves or another person
+
+Team members with outstanding changes requested reviews may dismiss their own reviews.
+
+Admins may dismiss outstanding reviews at their discretion, but should only do so if the team member cannot be contacted within a reasonable time-frame. Conflicts should *not* be resolved by an admin, but use 
+
+## Final Comment Period (FCP)
+- Begins immediately after the above criteria are met
+- Lasts 2 days for small implementation changes, 5 days for major refactors, and 10 days for new API introductions or RFCs. Bugfixes have no FCP and are considered mergable immediately.
+- May be ended at any time by a team member if they have concerns.
+
+
+## Conflicts
+If the Loader team cannot agree on whether to merge a PR or deny it, there are steps they can take to avoid decision paralysis. This should be a rare process--teams are encouraged to resolve these disagreements through talking the details out, but sometimes that isn't an option.
+
+In the event the disagreement is over API structure, the PR may, at the Loader team's option, be closed and an RFC be created that defines the feature.
+
+In the event the disagreement is over implementation details, or would otherwise not benefit from being forced through the long RFC process, the voting method defined below can be used.
+
+### Voting
+A vote is complete when one of the following is true:
+- Ten days has passed and the number of votes for one disposition is greater than the number of votes for the other disposition.
+- An absolute majority has been formed for one disposition.
+    - This means that even if all of the team members who have not yet voted suddenly voted for the currently losing side, the end result would not change. For example, if 7 out of a team of 10 have voted yes, the remaining 3 team members could not change the outcome of the vote.
+
+In the event of a tie, the vote is opened to the Admin Board and their votes count as 3. If the vote is still tied, the vote fails (the policy is not implemented, the PR is not closed, etc.)
```

Comment:
This does directly contradict [the voting process for the organization](https://github.com/QuiltMC/rfcs/blob/master/community/0006-governance.md#voting), which *might* be fine, but should be noted. Specifically, the general organizational voting process states that members of any supervising teams (up to an including the admin board) can participate in any votes of child teams. It's only when the vote is a tie that their weight is increased.